assets/base.css

```diff
@@ -209,15 +209,15 @@ html.no-js .no-js-hidden {
 }
 
 .element-margin {
-  margin-top: 5rem;
+  margin-top: calc(5rem + var(--page-width-margin));
```

Comment:
@ludoboludo I think it actually makes sense and here are my thoughts 🤔 

1) We are keeping it consistent and if the values change we only have to change it in one place.
2) these templates can have sections added to them so having the correct spacing makes sense.

Does that make sense? If anyting it would make sense to have article page and search page have custom spacing since this is a global class

Reply:
Yeah I can see why that's good. The overall feel of the site is more spaced out not only on the home page.